application/views/main/footer.php

```diff
@@ -1,2 +1,2 @@
 <?php
-echo tr('About {0}', NULL, '<a style="text-decoration: underline;grm" href="#" id="about_button">Kalkun '.$this->config->item('kalkun_version').'</a>');
+echo tr('About {0}', NULL, '<a style="text-decoration: underline;grm" href="javascript:void(0);" id="about_button">Kalkun '.$this->config->item('kalkun_version').'</a>');
```

Comment:
Wouldn't this require change in all language files?

Also on another note can we avoid having html inside translations?

Reply:
No, the language file has `About {0}`
The 3rd parameter is the one that repllaces  `{0}`

> Also on another note can we avoid having html inside translations?
It is there for historical reasons. I don't like it either.